src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Component.targets

```diff
@@ -210,12 +211,12 @@ Copyright (c) .NET Foundation. All rights reserved.
          KeyFile="$(KeyOriginatorFile)"
          LangVersion="$(LangVersion)"
          LinkResources="@(LinkResource)"
-         MainEntryPoint="$(StartupObject)"
          ModuleAssemblyName="$(ModuleAssemblyName)"
          NoConfig="true"
          NoLogo="$(NoLogo)"
          NoStandardLib="$(NoCompilerStandardLib)"
          NoWin32Manifest="$(NoWin32Manifest)"
+         NullableContextOptions="$(NullableContextOptions)"
```

Comment:
I'm tempted to leave this out. If nullability is enabled most Razor declaration files will fail to compile with nullability errors. @rynowak @pranavkm thoughts?

Reply:
For declarations? Yeah leave it out.